<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>URDialog</class>
 <widget class="QDialog" name="URDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transmit UR</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSend">
      <attribute name="title">
       <string>Send</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="URWidget" name="widgetUR" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btn_loadFile">
           <property name="text">
            <string>Load file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_loadClipboard">
           <property name="text">
            <string>Load from clipboard</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabReceive">
      <attribute name="title">
       <string>Receive</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QrCodeScanWidget" name="widgetScanner" native="true"/>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_clipboard">
         <property name="text">
          <string>Copy to clipboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_file">
         <property name="text">
          <string>Save to file</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QrCodeScanWidget</class>
   <extends>QWidget</extends>
   <header>qrcode/scanner/QrCodeScanWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>URWidget</class>
   <extends>QWidget</extends>
   <header>qrcode/scanner/URWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>URDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>URDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
